io.catenax.quality_task_attachment/2.0.0/QualityTaskAttachment.ttl

```diff
@@ -0,0 +1,172 @@
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.quality_task_attachment:1.0.0#> .
+
+:QualityTaskAttachment a samm:Aspect ;
+   samm:preferredName "Quality task attachment"@en ;
+   samm:description "Additional data for quality task"@en ;
+   samm:properties ( :qualityTaskId :relatedModelType :files ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:qualityTaskId a samm:Property ;
+   samm:preferredName "Quality Task ID"@en ;
+   samm:description "A unique quality task identifier where these list of parts analysis belong to."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "BPN-811_2022_000001" .
+
+:relatedModelType a samm:Property ;
+   samm:preferredName "relatedModelType"@en ;
+   samm:description "Indicates which aspect model these additional files are to be added to"@en ;
+   samm:characteristic :QualityModelType .
+
+:files a samm:Property ;
+   samm:preferredName "files"@en ;
+   samm:description "A list of files within the ZIP-folder, which are needed for a specific quality task"@en ;
+   samm:characteristic :ListOfFiles .
+
+:QualityModelType a samm-c:Enumeration ;
+   samm:preferredName "QualityModelType"@en ;
+   samm:description "Enumeration of released quality aspect models"@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "fleet.claim_data" "fleet.diagnostic_data" "manufactured_parts_quality_information" "parts_analyses" "quality_task" "vehicle.product_description" ) .
+
+:ListOfFiles a samm-c:List ;
+   samm:preferredName "List of files"@en ;
+   samm:description "List of files attached to a quality task"@en ;
+   samm:dataType :File .
+
+:File a samm:Entity ;
+   samm:preferredName "File"@en ;
+   samm:description "One file attached to the quality task"@en ;
+   samm:properties ( :fileName [ samm:property :schema; samm:optional true ] :filePath :sizeInKb :fileDescription :fileExtension ) .
+
+:fileName a samm:Property ;
+   samm:preferredName "file name"@en ;
+   samm:description "The name of the described file"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Histogramm_data.csv" .
+
+:schema a samm:Property ;
```

Comment:
Property and the referenced Characteristic should not have the same name

Reply:
Changed naming